website/integrations/services/mautic/index.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Integrate with Mautic
+sidebar_label: Mautic
+support_level: community
+---
+
+## What is Mautic
+
+> Mautic provides free and open source marketing automation software available to everyone. Free email marketing and lead management software.
+>
+> -- https://mautic.org/
+
+## Preparation
+
+The following placeholders are used in this guide:
+
+- `mautic.company` is the FQDN of the Mautic installation.
+- `authentik.company` is the FQDN of the authentik installation.
+
+:::info
+This documentation lists only the settings that you need to change from their default values. Be aware that any changes other than those explicitly mentioned in this guide could cause issues accessing your application.
+:::
+
+### Prepare the certificates
+
+Mautic and authentik both require an X.509 certificates.
+Mautic requires the key to be in PKCS#1-format specifically (and thus RSA).
+
+To avoid changing certificates in authentik (e.g., if they are present in PKCS#8), we generate a new one:
+
+1. Go to certificates and click on **Generate** and use the following values:
+    - **Common Name**: `Mautic Self-signed Certificate`
+    - **Private key Algorithm**: `RSA`
+2. Click on the drop-down arrow left of the newly generated certificate and **Download certificate** to `certificate.pem` and **Download Private key** to `private_key.pem`.
+3. Make sure that the `private_key.pem` is in PKCS#1 format. Therefore, open it in a text editor and check that the header footer contain `RSA`:
+    ```diff
+    - -----BEGIN PRIVATE KEY-----
+    + -----BEGIN RSA PRIVATE KEY-----
+    ```
+    and
+    ```diff
+    - -----END PRIVATE KEY-----
+    + -----END RSA PRIVATE KEY-----
+    ```
+    If the key contains `RSA` in header and footer, the certificate preparation is complete and you can skip steps 4 to 6.
+4. Add `RSA` after `BEGIN`/`END` in `private_key.pem` as shown in step 3 and save the file as `private_key_new.pem`.
+5. Regenerate the X.509-certificate by first creating a signing request:
+
+    ```sh
+    openssl req -new -key private_key.pem -out request.csr
+    ```
+
+    This will prompt to enter values for the certificate which you can choose freely. For some, you can use authentiks generated values:
```

Comment:
```suggestion
    This will prompt to enter values for the certificate which you can choose freely. For some, you can use authentik's generated values:
```